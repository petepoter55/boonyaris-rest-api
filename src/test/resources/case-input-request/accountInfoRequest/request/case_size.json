{
  "userId": "12345678901",
  "phone": "123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
  "birthDate": "1234567890112345678901",
  "citizenId": "123456789012345678901234567890123456789012345678901234567890",
  "province": "123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
  "district": "123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
  "createBy": "1234567789012345677890",
  "updateBy": "1234567789012345677890",
  "delFlag": true
}