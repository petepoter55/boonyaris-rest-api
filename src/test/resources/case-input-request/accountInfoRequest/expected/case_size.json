[
  "userId: size must be between 1 and 10",
  "phone: size must be between 0 and 25",
  "birthDate: size must be between 0 and 20",
  "citizenId: size must be between 0 and 50",
  "province: size must be between 0 and 150",
  "district: size must be between 0 and 150",
  "createBy: size must be between 0 and 20",
  "updateBy: size must be between 0 and 20",
  "birthDate: Expected format YYYY-MM-DD"
]